app/web/pages/[...slug].tsx

```diff
@@ -6,21 +6,16 @@ import { AUTH, GLOBAL, NOTIFICATIONS } from "i18n/namespaces";
 import { GetStaticPaths, GetStaticProps } from "next";
 import nextI18nextConfig from "next-i18next.config";
 import { serverSideTranslations } from "next-i18next/serverSideTranslations";
-import { getAllMarkdownPathsWithLocales } from "utils/markdownPages";
 
 async function getMarkdownPageBySlug(
   slug: Array<string>,
 ): Promise<MarkdownPageProps> {
   const md = await import(`markdown/${slug.join("/")}.md`);
-  return {
-    slug,
-    frontmatter: md.attributes,
-    content: md.html,
-  };
+  return { slug, frontmatter: md.attributes, content: md.html };
 }
 
 export const getStaticPaths: GetStaticPaths = () => ({
-  paths: getAllMarkdownPathsWithLocales(),
+  paths: [],
```

Comment:
We talked about this a few months ago on Slack, finally doing it. Especially as we don't translate the markdown files yet, loading a billion copies of them for different locales was showing red scary numbers at build time.